### ISSUE TITLE
Link to "Previous version" leading to current version

### ISSUE THREAD

**Issue by plannero** [w3c/wcag2ict#537]
In the latest WCAG2ICT version (dated October 08 2024 and available at https://www.w3.org/TR/wcag2ict-22/) there's a link in the "More details about this document" section labelled "Previous version". This link is https://www.w3.org/TR/wcag2ict/

The link, as far as I can tell, leads back to the same version of the document (again dated October 08 2024). So I am unable to easily find older versions of the specification.

Shouldn't "Previous version" lead to an older version of the document? Such as, for example an earlier draft, eg https://www.w3.org/TR/2024/DNOTE-wcag2ict-22-20240702/, or the last stable version before the current one?

**Comment by maryjom** [comment]
A link to the older version can be found in the [Background](https://www.w3.org/TR/wcag2ict/#background) section.

**Comment by maryjom** [comment]
And if you're looking for previous publications of this updated Note, use the link under the History: https://www.w3.org/standards/history/wcag2ict-22/ and you can find all of the published versions during the development of latest Note.

**Comment by maryjom** [comment]
@daniel-montalvo It is confusing that the previous version is linking back to the current one. Should it be pointing to the 2013 Note or to the last public draft of the latest Note?  Not sure what the convention is for Group Notes rather than specifications.
